charts/kube2iam/_kube2iam/daemonset.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: kube2iam
+  labels:
+    app: kube2iam
+spec:
+  template:
+    metadata:
+      labels:
+        name: kube2iam
+    spec:
+      hostNetwork: true
+      containers:
+        - image: jtblin/kube2iam:latest
+          name: kube2iam
+          args:
+            - "--base-role-arn=arn:aws:iam::593291632749:role/"
```

Comment:
Initially I thought that too, but I think we might well end up deploying other services that use roles that won't be user-based, so it seems best to allow any role to be specified regardless of path.